Automattic/wp-calypso
# PostTypeList: Fix "view" in "All My Sites"

This PR fixes an issue where the "view" menu item wouldn't show a preview while in "All My Sites" mode while using 

The issue only exists behind the feature flag `posts/post-type-list`.

To test:

* `ENABLE_FEATURES=posts/post-type-list npm start` (or test via calypso.live using the link below)
* Visit /posts and verify the acceptance criteria
* Verify that the new tests pass:
  * `npm run test-client client/state/ui/preview/test/reducer.js`
  * `npm run test-client client/state/ui/preview/test/selectors.js`


Acceptance criteria
1.
- Given a list of posts in All My Sites
- When the View action for a post is selected from the actions menu
- Then a preview of the post is shown

2.
- Given a list of posts for a single site
- When the View action for a post is selected from the actions menu
- Then a preview of the post is shown

3.
- Given a list of posts for single non-HTTPS Jetpack site
- When the action menu is shown for a post
- Then the View action opens the post in a new tab
